src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -1463,10 +1463,7 @@ static bool filter_chunks(const ObjectRepositoryItem* item)
                     break;
                 }
             }
-            if (_filter_flags & (1 << (gRideCategories[rideType] + _numSourceGameItems)))
-                return true;
-
-            return false;
+            return (_filter_flags & (1 << (gRideCategories[rideType] + _numSourceGameItems))) != 0;
```

Comment:
I prefer the old version, I think that is more readable than the new one.

Reply:
Revert to original or use `return (_filter_flags & (1 << (gRideCategories[rideType] + _numSourceGameItems)));`?

Reply:
To the original.